.circleci/config.yml

```diff
@@ -19,6 +19,14 @@ parameters:
     type: string
     # solbuildpackpusher/solidity-buildpack-deps:ubuntu2204.clang-8
     default: "solbuildpackpusher/solidity-buildpack-deps@sha256:2662376fe0e1ec2d346495a19a6d64508c1048d5a7325d8600c33c343fa64a0f"
```

Comment:
Why was #15321 even merged without having a matching PR updating CI that passes all checks? We're in a really weird state now where the dockerfile source does not match what we're running and also it blocks us from doing any other changes to the images that we might need in the meantime (at least without reverting them). Good thing that we apparently did not need any so far.

Unless these are going to be bumped or removed **very** soon, we should really revert these images (with a version bump to avoid reusing the number).

Reply:
This you mean? https://github.com/ethereum/solidity/pull/15320

We had a ton of problems with the new evmone bump (and still do for ossfuzz) while you were off.

Reply:
Yes, I mean why did we decide to merge #15321 even though #15320 wasn't passing CI? I remember that there were problems and that's fine, just saying that in that case the right procedure is to hold off on merging both.